runtime/doc/eval.txt

```diff
@@ -2826,6 +2839,74 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+call_async({func}, {args}, {opts}[, {callback}])     	*call_async()*
+		Asynchronously call function {func} with the items in |List|
+		{args} as arguments.
+		{func} can either be a |Funcref| or the name of a built-in
+		function (|functions|).
+		Returns the |job-id| of the child process running the
+		function. (see |call_wait()|).
+		When the called function returns, {callback} is invoked with
+		the return value as an argument.  Example: >
+			call call_async(
+			      \ 'nvim_grep',
+			      \	['int', '**/*.h', v:false], {},
+			      \	{result ->
+			      \	 nvim_command('echo len(result)')})
+<
+		{opt} is a dictionary specifying context information to carry
+		to the child process, it can have the following keys (all
+		default to |FALSE| except for varscope):
+			varscope  outermost scope to include (default: "none")
+				  "none":   no surrounding scope accessible
+				  "local":  |a:var| and |l:var|
+				  "script": |a:var|, |l:var|, and |s:var|
+				  "global": caller scope (all vars accessible)
+			wvars	  include |w:var| variables
+			bvars	  include |b:var| variables
+			tvars	  include |t:var| variables
+			gopts	  include global options
+			wopts	  include window options
+			bopts	  include buffer options
+			arglist	  include |arglist|
+			qflist	  include |quickfix| list
+			loclist	  include |location-list|
+			regs	  include |registers|
+			marks	  include |mark|s
+			btext	  include buffer text
+
+							*call_parallel()*
+call_parallel({count}, {func}, {argslist}, {opts}[, {callback}])
```

Comment:
Re API `count` could go into `opts` and default to all cores (`v:cores`) maybe?

Reply:
Hmmm...yeah this is a sane default